.env.example

```
@@ -24,20 +12,16 @@ INTERNAL_AUTH_KEY="bwgybgidbsi-4784gyfgs-475hhkdsbfs-bwgybgidbsi-4784gyfgs-475hh
 R2_BUCKET_NAME="your-org-name-userdata"
 R2_SECRET_ACCESS_KEY="279y45g79tgbjsbfhbsiufhbs89hg487hsiufs"
 TURSO_AUTH_TOKEN="dhbBDSUGFBSUYGBSIUGRBIGBSIYBS7495w8y97w.bivsb7478wGYIFGDSUGFS48y39975y3gtyugysjgs7ugu.BHFBSYUFBSV476guysfuw78gfuy3buybfshbfYUFSGVBYFIBVEIUSBVJSBANIBA74y597bvs6gfusg&gsyjgbfshgbyIGBYUS"
 TURSO_DATABASE_URL="libsql:hackkit-orgname.turso.io"
 UPSTASH_REDIS_REST_TOKEN="BKFSHBSIY7g9347385gBJFDBSJYFGBSIYFUSIBISUYHGOSHIYtusvufdbsfiubfsibYBG"
 UPSTASH_REDIS_REST_URL="https://giant-worm-78102.upstash.io"
+
+SMTP_HOST="mailserver.hackkit.org"
+SMTP_PORT="25"
+SMTP_SECURE="true"
+SMTP_USER="user"
+SMTP_PASS="password"
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)